.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -16,11 +16,11 @@ Please include a summary / description of the change and which issue it fixes.
 
 
 ## Checklist PR Raiser
- - [ ]  I've made every effort to commit high quality, clean code and I have executed relevant static analyses to be sure
```

Comment:
can this file be reverted as dan commented, this is picked up when creating a pr and can be changed at that point which won't affect this file